styles.css

```diff
@@ -133,11 +134,28 @@ button:hover {
   text-align: center;
 }
 
+.new-idea-title {
+  width: 100px;
+  align-self: center;
+}
+
 .disabled {
   background-color: lightgray;
   color: grey;
   cursor: not-allowed;
 }
 
+.delete-container {
+  display: flex;
+  background-color: #1F1F3C;
+  margin: -10px;
+  height: 50px;
+  justify-content: flex-end;
+}
```

Comment:
@ben-rosner-williamsburg Not sure why, but this fella seems to be sticking out of the new idea container. Maybe a smidge more tweaking to get it to fit to the container properly? 
<img width="431" alt="Screenshot 2023-09-14 at 9 16 24 AM" src="https://github.com/ericahagle/ideabox/assets/133910120/ce46ac1c-1ad0-4f9f-823e-8264e7d7bde3">

Reply:
@ericahagle I will look into that now! Thanks!

Reply:
@ericahagle I just made a new commit here. Let me know if that looks good!

Reply:
Much bettah! 🍬 
<img width="427" alt="Screenshot 2023-09-14 at 10 06 22 AM" src="https://github.com/ericahagle/ideabox/assets/133910120/8fd54cf5-e489-4af7-bb55-997096b46a93">